### ISSUE TITLE
coding-standard.xhtml mislinks to twisted.python.compat

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3382]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3382|
|Type|defect|
|Created|2008-08-04 14:26:04Z|
|Branch|https://github.com/twisted/twisted/tree/mislink-compat-3382|

The _Standard Library Extension Modules_ section talks about and links to `twisted.compat.set` and `twisted.compat.frozenset`.

<details><summary>Searchable metadata</summary>

```
trac-id__3382 3382
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_mislink_compat_3382 branches/mislink-compat-3382
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1217859964000000 1217859964000000
changetime__1222276299000000 1222276299000000
version__None None
owner__ 
cc__exarkun cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [24897]) Branching to 'mislink-compat-3382'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24917]) Merge mislink-compat-3382: coding standard mislinked to twisted.python.compat

Author: thijs
Reviewer: exarkun
Fixes: [#3382](https://github.com/twisted/twisted/issues/3382)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
